### PR TITLE
[TRAFODION-2431] python installer - add python lib dependencies

### PR DESCRIPTION
**Pull request by mkby**
@svarnau, please help to review.

### CODE REVIEW

File: licenses/lic-pyinstall-src [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
++++++++++++++++++++++++++++++
+
+MIT-Expat for files in:
+    incubator-trafodion/installer/python-installer/scripts/httplib2/iri2uri.py
+    incubator-trafodion/installer/python-installer/scripts/httplib2/__init__.py
+
```
Comment (svarnau) [inline licenses/lic-pyinstall-src]: same as above, start path at installer, you can leave off the incubator-trafodion directory.

File: licenses/lic-pyinstall-src [hunk 2]
```diff
@@ -0,0 +1,63 @@
+===========================================================================
+The python installer dependencies bundles source with BSD-like and
+MIT-like licenses:
+
++++++++++++++++++++++++++++++
+
+BSD-3 clause for files in:
+    incubator-trafodion/installer/python-installer/prettytable.py
+    incubator-trafodion/installer/python-installer/scripts/httplib2/socks.py
+
```
Comment (svarnau) [inline licenses/lic-pyinstall-src]: leave off incubator-trafodion directory. The workspace can have different name.

File: .rat-excludes [hunk 3]
```diff
@@ -166,8 +166,14 @@ VariableLengthPKTest.java
...
 mod_cfgs.json
 prompt.json
 script.json
 version.json
+
+# pyinstaller dependencies
+prettytable.py
+socks.py
+iri2uri.py
+__init__.py
```
Comment (svarnau) [inline .rat-excludes]: These files all have headers. RAT is only confused by two of them (prettytable & socks). So the others should NOT be listed in this file.

Also, comment here should not just state what they are, but why they should be excluded from check -- they have BSD-3 license that RAT does not recognize.

**Comment by mkby** [comment]
@svarnau，please review again, thanks

**Pull request merged**